exercise/api/custom_serializers.py

```diff
@@ -49,10 +49,20 @@ def submission_url(submission_id):
                 'submission_id': submission_id
             }, request=request)
 
+        def submission_obj(submmission_cached):
```

Comment:
typo `submmission_cached`

Reply:
Also, these might not be the best names for the function and the variable. Feel free to rethink them once or twice.